common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -6,11 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.4" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.3" />
-    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.0.0-preview.4" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.0.0-preview.3" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.6.0" />
+    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.5" />
```

Comment:
Any reason to bump these to the next prerelease as opposed to GA?

Reply:
These should really be the old values... I want to keep the bumping separated and in a different PR